lisa/tools/find.py

```diff
@@ -22,6 +22,7 @@ def find_files(
         start_path: PurePath,
         name_pattern: str = "",
         path_pattern: str = "",
+        find_type: str = "",
```

Comment:
`type` should be enough. We already have `find` prefix in function name

Reply:
Also, this should be Optional[str] datatype

Reply:
Have changed to 'type'.  The datatype str is enough I think.